vertx-vaadin-flow-parent/vertx-vaadin-flow/src/main/java/com/github/mcollovati/vertx/web/serialization/SerializableHolder.java

```diff
@@ -24,13 +24,16 @@
 
 import io.vertx.core.buffer.Buffer;
 import io.vertx.core.shareddata.impl.ClusterSerializable;
+import lombok.NonNull;
 
-public class SerializableHolder implements ClusterSerializable {
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
 
-    private transient Object object;
+public class SerializableHolder implements ClusterSerializable, Serializable {
```

Comment:
This class was not mean to be used directly by standard serialization, but only within vertx session.
Have you found any problem on having it not Serializable?